docs/guides/preparing/local-server-setup.mdx

```diff
@@ -0,0 +1,302 @@
+  />
+</div>
+
+You also need to mark the script file as executable. Start by opening Terminal, type the following command below (but **do not press** Enter yet) and drag the file into the window.
+The file path will appear and highlight itself once this is done, then you can press Enter.
+
+```bash
+chmod a+x <path to your start.command file>
+```
+
+<div className="text--center">
+  <ImageSwitcher loading="lazy"
+    lightImageSrc="/img/guides/local-server-setup/mac-sportpaper-c3-light.png"
+    darkImageSrc="/img/guides/local-server-setup/mac-sportpaper-c3-dark.png"
+  />
+</div>
+
+You must always double-click the `start.command` file to launch the server, not the .jar file.
+
+### Downloading PGM
+
+You have two options when it comes to downloading PGM. You can download the stable version of PGM from the [Downloads](/downloads) page or you can download the development builds from its [GitHub repository](https://github.com/PGMDev/PGM/actions?query=actor%3AElectroid+branch%3Adev+is%3Asuccess).
+If you are working on a map that will utilize features introduced in pre-release versions of PGM, we recommend downloading the development build.
+
+Once the download is done, you must put `PGM.jar` inside the `plugins` folder (create if it does not exist), which can be found in the folder that contains the `SportPaper.jar` file.
+
+<div className="text--center">
+  <ImageSwitcher loading="lazy"
+    lightImageSrc="/img/guides/local-server-setup/mac-pgm-c1-light.png"
+    darkImageSrc="/img/guides/local-server-setup/mac-pgm-c1-dark.png"
+  />
+</div>
+
+### Starting the Server
+
+Once you are done with the initial setup, you can double-click `start.command` to let SportPaper create the necessary files for Minecraft and PGM.
+You can now join the server from your Minecraft client at `localhost` or `127.0.0.1:25565`.
+
+Also see [Additional Setup](#additional-setup) if you would like to make the most use out of your newly-created server.
+
+<div className="text--center">
+  <ImageSwitcher loading="lazy"
+    lightImageSrc="/img/guides/local-server-setup/mac-serversetup-c1-light.png"
+    darkImageSrc="/img/guides/local-server-setup/mac-serversetup-c1-dark.png"
+  />
+</div>
+
+## Linux
+
+This guide uses commands that are available for Debian-based distributions.
+Please keep in mind that the following process may differ depending on your distribution. If necessary, consult your distribution's documentation or user manual.
+Otherwise, we trust that you know what you are doing.
+
+### Installing Java
+
+We recommend using either Java 8 or Java 11. For this guide, we will be using [OpenJDK](https://openjdk.org/). 
+However, you may choose to use [Amazon Corretto 11](https://aws.amazon.com/corretto/) or [Eclipse Adoptium](https://adoptium.net/) as well.
+If you already have a copy of Java 8 or Java 11 installed, you may skip this step.
+
+```bash
+sudo apt update && sudo apt install openjdk-11-jdk
```

Comment:
```suggestion
$ sudo apt update && sudo apt install openjdk-11-jdk
```

Reply:
i would avoid adding the `$`, if you know what you're doing it probably won't help, and if you don't know what you're doing and are just copy-pasting the commands in, will prevent you from running them